CMakeLists.txt

```diff
@@ -39,11 +39,14 @@ endif()
 option(INSTALL_HEADER_ONLY "Install SYCL-BLAS as a header only library" OFF)
 
 set(BUILD_SHARED_LIBS ON CACHE BOOL "")
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
We may want to remove the line where we set `CMAKE_CXX_STANDARD` again below line 80 for `MSVC`?